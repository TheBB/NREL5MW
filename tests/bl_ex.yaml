Re: 10000.0
ahead: 0.0
behind: 4.0
d_join: 0.2
d_tip: 0.006
debug: false
format: IFEM
in_antihub: in
in_hub: in
in_slip: in
join_adds: 0
join_index: 4
len_te: 0.02
len_te_cyl_fac: 50.0
length: 10.0
length_mode: extruded
mesh_mode: blade
n_ahead: 2
n_back: 29
n_base: 0
n_behind: 36
n_bndlayer: 4
n_circle: 72
n_front: 41
n_length: 160
n_sides: 2
n_square: 10
n_te: 10
order: 4
out_antihub: antihub
out_hub: hub
out_slip: slip
paramfile: examples/naca_wingdef.yaml
p_ahead: 1
p_behind: 1
p_inner: 2
p_length: 20
p_rigid: 0
p_sides: 1
radius: 10.0
rho: 1.0
sides: 0.0
slip_antihub: antihub
slip_hub: hub
smoothing: true
velocity: 1.0
walldistance: false
wing_antihub: antihub
wing_hub: hub
