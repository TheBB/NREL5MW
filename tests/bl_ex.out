geometry.g2
patch 1
<type 'GoTools.Surface'>
kmult dir 1: 4 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4
kmult dir 2: 4 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4
